tools/msvs/msi/i18n/fr-BE.wxl

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
why only add fr-BE and not fr-FR ? is it intended ?
The PR title don't mention specifically Belgium